packages/flutter/lib/src/material/about.dart

```diff
@@ -1396,7 +1397,10 @@ class _MasterDetailScaffoldState extends State<_MasterDetailScaffold>
               ),
             ),
           ),
-          body: _masterPanel(context),
+          body: Align(
```

Comment:
I think this can be simplified and the `directionality` variable removed:

```
          child: Align(
            alignment: AlignmentDirectional.centerStart,
```

Reply:
I didn't know that, awesome